pkg/cli/server.go

```diff
@@ -201,7 +201,8 @@ func newServerService(c *config.Server) (*serverService, error) {
 	svc.scrapeManager = scrape.NewManager(
 		svc.logger.WithField("component", "scrape-manager"),
 		ingester,
-		defaultMetricsRegistry)
+		defaultMetricsRegistry,
+		svc.config.DisableCumulativeMerge || !svc.config.RemoteWrite.Enabled)
```

Comment:
@korniltsev I have a question about this.

First of all, I rewrote the original boolean logic, because it was too confusing.

So this is what we have now:
![Screenshot 2022-12-15 at 4 21 43 PM](https://user-images.githubusercontent.com/662636/207993937-72eee489-64c5-4c2b-8b87-a51ed9c0c33c.png)

But even after doing that I don't think I understand why if remote write is enabled we should be doing force merges. Could you explain?